integration-test/src/main/java/com/sequenceiq/it/cloudbreak/util/ClouderaManagerUtil.java

```diff
@@ -79,7 +79,7 @@ public static StackTestDto checkClouderaManagerUser(TestContext testContext, Sta
                 throw new TestFailException("Requested user details are not valid " + userDetails);
             }
         } catch (Exception e) {
-            LOGGER.error("Can't get users' list at: %s or test user is not valid with %s", cmClient.getBasePath(), userDetails);
+            LOGGER.error(String.format("Can't get users' list at: %s or test user is not valid with %s", cmClient.getBasePath(), userDetails));
```

Comment:
Please don't use the String.format with a logger, it kills the option to do runtime optimization based on log level. Use instead this version: `LOGGER.error("Can't get users' list at: {} or test user is not valid with {}", cmClient.getBasePath(), userDetails)`.